### ISSUE TITLE
Get-InstalledPSResource should be Get-PSResource

### ISSUE THREAD

**Issue by ThomasNieto** [PowerShell/PSResourceGet#1072]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PowerShellGet)
- [X] Search the existing issues.

### Steps to reproduce

Creating this issue because my comment was not addressed in #919 about `Get-PSResource` rename to include `Installed`.

Latest MS reasoning https://github.com/PowerShell/PowerShellGet/issues/919#issuecomment-1419636861:
> We had an internal discussion about this, and want to make it analogous to V2 cmdlets where we have Get-InstalledModule (from the PSGet side) and Get-Module (from PowerShell side) and reduce confusion between their different outputs. Get-InstalledModule (and in V3 Get-InstalledPSResource) would get resources installed via PowerShellGet on your machine, not all of them as Get-Module does. We will also look into an alias for the command.

My comment https://github.com/PowerShell/PowerShellGet/issues/919#issuecomment-1419871881
> There is no overlap/confusion between Get-Module and Get-PSResource because the noun has changed to PSResource. So why are we trying to keep the v2 convention with "Installed" as a prefix? I'm trying to understand the reasoning as "Installed" is redundant as it's already implied that it's installed on the system with the Get verb. If it wasn't already installed or locally present on the system then it would have a different verb.

Issue #919 and PR #1071.

I would like to know why the team thinks there will be added value by adding `Installed` when the noun is `PSResource` not `Module` which was the issue with v2.

### Expected behavior

```console
Get-PSResource
```

### Actual behavior

```console
Get-InstalledPSResource
```

### Environment data

```powershell
7.4-preview3
beta21
```

**Comment by ThomasNieto** [comment]
@SydneyhSmith can we get an update on why MS thinks adding `Installed` as a prefix creates value to `Get-PSResource`? In my comment rebutting the change had others like @Jaykul agreeing with me.

Here are some other points as well:

* Longer cmdlet names creates user experience difficulties especially with those who do not use tab completion.
* Users expect modules to have the same noun for cmdlets, this breaks that convention.
* Users can no longer use `*-PSResource` with tab completion or `Get-Command -Noun PSResource` to get all the verbs associated with that noun.
* The only reason that v2 had `Installed` as a prefix for `Get-InstalledModule` is because `Get-Module` was already used and something else had to be used. By using `Get-InstalledModule` caused user confusion since the two cmdlets didn't use the same output. This is no longer an issue because v3 uses a completely different noun `PSResource`. There is no user expectation that the output between `Get-Module` and `Get-PSResource` would be the same. 

If there is something I'm missing please let me know but right now the pros/cons are pointing to using `Get-PSResource`.

**Comment by SydneyhSmith** [comment]
@ThomasNieto yes happy to put together a little write up, want it to be thoughtful and have a really full calendar today but will plan to get it up early next week

**Comment by SydneyhSmith** [comment]
Talked again with the team and our reasoning was: 

- Clarity on what it is getting--all installed resources
- Parity with Get-InstalledModule
- Leaves the door open for a future cmdlet of Get-PSResource which would just show loaded PSResources like Get-Module does

Totally appreciate that might not be satisfying, and appreciate your reasoning as well, going to leave the issue open for discussion

**Comment by ThomasNieto** [comment]
@SydneyhSmith while it is nice that you're leaving this open for more feedback. It would have been better to have implemented this per the RFC since most of the feedback has been given there and there was no negative feedback in the RFC regarding `Get-PSResource` in regards to the first two points you have.

> Clarity on what it is getting--all installed resources

Has there been any user feedback during the RFC or previews that `Get-PSResource` was confusing in regards to what the cmdlet did? Was there any user feedback to use the v2 naming convention? I have not seen any.

> Parity with Get-InstalledModule

Why does this matter? The rest of the cmdlets behave completely differently than v2. I would agree with the team if there is a large precedence in core cmdlets and community cmdlets using `Installed` as a well known prefix.

> Leaves the door open for a future cmdlet of Get-PSResource which would just show loaded PSResources like Get-Module does

Leaving the merits of showing loaded resources aside. What benefit is there in having another cmdlet compared to adding a parameter or parameter set?

`Get-Module` is a more used and better user experience precedent for `Get-PSResource`. Lets see how that behaves. `Get-Module` with no parameters returns loaded modules. If `-ListAvailable` is used it returns "installed" modules. Now lets compare to this fictional feature of showing loaded PSResources. There are two options (Installed and Loaded) for default behavior and one that is more common should be default.

Lets say "installed" resources should be the default behavior then `Get-PSResource` returns those by default. Then there could be a `-Loaded` parameter to only return loaded resources. The single cmdlet accomplishes both features without any user confusion or difficulty. There is also the option of not even needing a separate parameter and `Loaded` could be a property. Then a view could show it by default or let it be sorted/filtered by the user.

Now lets discuss the merits of this fictional feature. There are currently only two resources: Modules and Scripts with a discussed feature of NuGet assemblies. Modules are already covered with `Get-Module` and scripts aren't loaded in the session. There is some merit to showing loaded assemblies but shouldn't that be a core cmdlet?

TLDR: `Installed` does not have the precedence in core cmdlets and is not a commonly used prefix in community modules. The parity in regards to v2 is not relevant since v3 is supposed to fix all the issues with v2 in regards to user experience. Then in regards to the fictional feature described there is not a need to have two separate cmdlets to accomplish both features when properties, parameters or parameter sets is used.

**Comment by SydneyhSmith** [comment]
@ThomasNieto just wanted to acknowledge that I appreciate the feedback, and am considering what our best options are at this point

**Comment by alerickson** [comment]
Resolved in #1216
